.bazelrc

```diff
@@ -1,3 +1,4 @@
 build --verbose_failures=true
 common --announce_rc
 build --explain=explain.log --verbose_explanations
+build --incompatible_default_to_explicit_init_py
```

Comment:
mirror what we have in mms--was needed to get the test script working. Also we might want the `enable_implicit_namespace_pkgs` attr of the pip_repository rule, but not sure.